### ISSUE TITLE
Bug introduced in 3.51 - LibStub() cannot be called

### ISSUE THREAD

**Issue by cbracht** [Rottenbeer/ItemRack#140]
Error when Logging in, Addon fails to load:

```
Message: Interface\AddOns\ItemRack\ItemRack.lua:157: attempt to call global 'LibStub' (a nil value)
Time: Tue Nov 24 17:23:33 2020
Count: 1
Stack: Interface\AddOns\ItemRack\ItemRack.lua:157: attempt to call global 'LibStub' (a nil value)
Interface\AddOns\ItemRack\ItemRack.lua:157: in main chunk

Locals: _ = nil
(*temporary) = nil
(*temporary) = "Masque"
(*temporary) = true
(*temporary) = "attempt to call global 'LibStub' (a nil value)"
```

Code where the error occurs (Line 157):
![image](https://user-images.githubusercontent.com/1489764/100123415-c3c09c80-2e7a-11eb-83a8-d73950801b4a.png)

**Comment by Rottenbeer** [comment]
@cbracht thanks for reporting it.
Do you have masque installed or not?
Can you provide a list of addons that you are using?
Did you try to delete the ItemRack settings in the WTF folder and does it fix the problem?

@pshurgal This might be a hint for the other problems you where already looking at?

**Comment by cbracht** [comment]
@Rottenbeer masque is not installed. This looks like a library some other addon is probably providing but is not present with my used addons. Deactivating all other addons or removing ItemRack settings does not fix the problem.

Fixed by exchanging L157 with
`local Masque = (LibStub and LibStub("Masque", true)) or (LibMasque and LibMasque("Button"))`

**Comment by Rottenbeer** [comment]
Yeah LibStub is provided by Ace and is bundled with Masque and probably more addons, that's why it's kinda random I guess.

**Comment by Rottenbeer** [comment]
Can you try to download this https://github.com/Rottenbeer/ItemRack/tree/libstub and check if it works?

**Comment by cbracht** [comment]
Works as expected for me (though I dont have another addon that provides LibStub)
Thank you and every other contributor for their great work!

**Comment by Rottenbeer** [comment]
Will be released on curseforge as 3.52 any minute.
